appium.js

```diff
@@ -73,12 +91,91 @@ function connectGetDefaultArgs () {
   });
 }
 
+function connectStartSession (win) {
+  ipcMain.on('start-session', () => {
+    let sessionWin = new BrowserWindow({width: 800, height: 600, webPreferences: {devTools: true}});
+    let sessionHTMLPath = path.resolve(__dirname, 'app', 'index.html#/session');
+    sessionWin.loadURL(`file://${sessionHTMLPath}`);
+    sessionWin.show();
+
+    // When you close the session window, kill the associated Appium session
+    let sessionWinID = sessionWin.webContents.id;
+    sessionWin.on('closed', () => {
+      killSession(sessionWinID);
```

Comment:
probably should show a spinner or something to the user while we're quitting the session